sys/net/network_layer/ng_sixlowpan/ng_sixlowpan.c

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include "kernel_types.h"
+#include "net/ng_netbase.h"
+#include "thread.h"
+#include "utlist.h"
+
+#include "net/ng_sixlowpan.h"
+#include "net/ng_sixlowpan/netif.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+static kernel_pid_t _sixlowpan_pid = KERNEL_PID_UNDEF;
+static char _sixlowpan_stack[NG_SIXLOWPAN_STACK_SIZE];
+
+void _handle_receive(ng_pktsnip_t *pkt)
```

Comment:
to be aligned with ipv6: just _receive() and _send()

(I thought about it to prefix this in IPv6 with _handle, too, but I think small is beautiful here.